Makefile.toml

```diff
@@ -129,3 +129,38 @@ description = "Clean up WASM build artifacts"
 category = "ICU4X WASM"
 command = "rm"
 args = ["-rf", "wasmpkg"]
+
+[tasks.bincode-clean]
+description = "Clean out the bincode data."
+category = "ICU4X Bincode"
+script_runner = "@duckscript"
+script = '''
+# Use duckscript, rather than a unix command, so that this works on Windows.
+rm ./resources/testdata/data/bincode
+'''
+
+[tasks.bincode-gen-testdata-no-clean]
+description = "Generate bincode testdata without removing the old data"
+category = "ICU4X Bincode"
+cwd = "components/provider_fs"
+command = "cargo"
+args = [
+    "run",
+    "--features", "export-bin",
+    "--features", "bincode",
+    "--features", "serialize_none",
+    "--",
+    "--cldr-tag", "38.1.0",
```

Comment:
I believe I had to choose a tag here, and arbitrarily chose the latest.

Reply:
Using a tag means that the CI needs to download 1 GB of data from GitHub each time.  That's flaky and prone to error.  Instead, since #513, which was checked-in a couple days ago, you can point directly to CLDR JSON in the ICU4X repo and remove the external dependency.

Reply:
This should be resolved now. It's using the checked in data.